community/saas-companies-engagement-procedure.md

```diff
@@ -0,0 +1,118 @@
+using the SPIFFE specification by designating responsible technical contacts.
+
+#### Guidelines
+
+##### Designation of Contacts
+
+Each SaaS company that directs customers to the SPIFFE Slack channel must 
+designate two technical contacts.
+
+These contacts will serve as the primary point of contact for addressing queries 
+and facilitating communication between the customers and the Spiffy community.
+
+##### Documentation of Contacts
+
+A dedicated folder for each company will be created in the SPIFFE repository
+under the `./community/saas-technical-contacts` directory.
+
+The folder will contain:
+
+* Names and contact information of the designated technical contacts.
+* Guidelines for interaction and support procedures.
+* Any relevant documentation provided by the company.
+
+### Interaction Protocols
+
+#### Objective
+
+To establish clear protocols for interactions between SaaS consumers and the 
+SPIFFE community to ensure a productive and respectful environment.
+
+#### Guidelines
+
+##### Communication Etiquette
+
+All interactions should be professional, respectful, and focused on technical 
+issues.
+
+Technical contacts are responsible for ensuring that their customers, and anyone
+who join the community through their reference adhere to the community 
+guidelines and maintain a positive environment.
+
+##### Issue Resolution
+
+Technical contacts should facilitate the resolution of technical issues by 
+providing detailed information and context.
+
+For complex issues, technical contacts should coordinate with SPIFFE maintainers 
+and contributors to find solutions.
+
+##### Feedback Mechanism
+
+SPIFFE Steering Committee encourages SaaS consumers to provide feedback on their 
+experience with the SPIFFE specification and the support received.
+
+SPIFFE Steering Committee will use this feedback to improve documentation, 
+support processes, and community guidelines.
+
+### Managing Disruptive Participants
+
+#### Objective
+
+To identify and manage participants who disrupt the community, ensuring that 
+the SPIFFE Slack channel remains a productive and positive space.
+
+#### Guidelines
+
+##### Identification of Disruptive Behavior
+
+Disruptive behavior includes, but is not limited to, spamming, harassment, and 
+consistently off-topic discussions.
+
+Technical contacts should monitor interactions and identify any disruptive 
+behavior from their customers.
+
+##### Initial Warning
+
+If a participant is identified as disruptive, a warning should be issued 
+outlining the behavior and the consequences of continued disruption.
+
+Technical contacts are responsible for communicating the warning to the 
+participant.
+
+##### Escalation Process
+
+If the disruptive behavior continues, escalate the issue to the SPIFFE 
+Steering Committe.
+
+The SSC will evaluate the situation and decide on appropriate actions, which may 
+include temporary or permanent removal from the Slack channel.
+
+##### Documentation and Reporting
```

Comment:
```suggestion
include temporary or permanent removal from the Slack channel.

If a particular company's customers have a mass habit of disruption or other issues, they will be evicted from channels and the list of community partners.

##### Documentation and Reporting
```

Reply:
Makes sense, maybe I'll alter the wording a bit. 
Will update and let here know.